ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-stable-4.13.yaml

```diff
@@ -191,6 +191,7 @@ tests:
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
       BASELINE_CAPABILITY_SET: None
+      CAPABILITIES_IN_CREDENTIALREQUEST: ImageRegistry MachineAPI
```

Comment:
How about rename the  var to `EXPECTED_CAPABILITIES_IN_CREDENTIALREQUEST` for easy reading.

Reply:
updated